cert-manager/cert-manager/base/services.yaml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
nit - even with multiple resources most manifests have just called it service.yaml

Reply:
Same here.